chapters/conclusion.tex

```diff
@@ -1 +1,217 @@
+associated with changes in resource capacity and requirements for the population
+under study. These adjustments revealed actionable insights into the
+most-impactful segments of the population. The most important of these results
+was demonstrating the futility of attempting to implement quick, blanket
+solutions for that population, such as only increasing resource capacity without
+improving patient well-being.
+
+
+\section{Contributions}\label{sec:contributions}
+
+This thesis has made novel contributions across each of its three principal
+themes: algorithm evaluation, clustering, and healthcare modelling. This section
+summarises these contributions with reference to their respective chapters.
+
+The EDO method introduced in Chapter~\ref{chp:edo} provided an example approach
+from a novel paradigm in which the objective performance of algorithms can be
+assessed by exploring the space in which `good' or `bad' datasets exist. The
+proposed paradigm expands the commonly used approach for evaluation where a
+method's quality is `confirmed' by taking a small number of benchmark datasets
+and comparing them with its contemporaries. By exploring the space of datasets,
+it was demonstrated that a more robust assessment can be made about a method or
+set thereof.
+
+Chapter~\ref{chp:kmodes} added to the growing body of literature where
+game-theoretic concepts are combined with machine learning techniques, of which
+clustering is included. In general, pursuits of this kind reformulate existing
+techniques to be mathematically fair. The initialisation presented in
+Chapter~\ref{chp:kmodes}, instead, incorporated game theory directly into an
+existing algorithm. In doing so, an improvement over the existing method was
+shown, using both traditional confirmation processes and the EDO method.
+
+The framework used in Chapter~\ref{chp:copd} contributed to healthcare modelling
+literature in three ways. First, the estimation of queuing parameters via the
+Wasserstein distance has expanded a relatively scarce aspect of queuing
+research. Second, by making COPD the subject of the methodology, the framework
+has added to a body of literature surrounding a condition that is vital to
+understand given its prevalence, as well as its links to deprivation and
+comorbidity. Lastly, the framework provided a solution to the common issue of
+data availability in modern operational research. By combining the various
+individual methods, valuable insights were extracted from a relatively
+unsophisticated data source, which is a result seldom seen in operational
+research.
+
+In addition to the work directly included in the chapters of this thesis, the
+research associated with this thesis has resulted in the production of numerous
+auxiliary research items. These include several well-developed pieces of
+research software, and a number of useful, publicly available datasets for
+clustering and healthcare modelling.
```

Comment:
Can we point at them (listed in brackets is fine, or perhaps this needs to be a standalone page with everything listed, I forget if you have that already?).

Reply:
I don't have one for the datasets, but I do list the DOI names for them in the introduction. The software packages are summarised in the dissemination.

Do you think another page would be helpful?

Reply:
It's probably fine to have those two spots in which case can you just point to them here? Something like:

```
research software (listed in \ref{dissemination}, and a number of useful, publicly available datasets (listed in Section \ref{}).
```

Reply:
Cool cool. Added in those references in ac10058